forge/forge/tvm_to_python.py

```diff
@@ -1964,6 +1964,9 @@ def make_parser_friendly_name(node, node_type):
                     if inp_shape[:2] == out_shape[:2] and inp_shape[2] * 2 == out_shape[2]:
                         input_names = [input_names[0], input_names[0]]
 
+                if json_graph["device"] == "tt" and node["name"] == "embedding" and len(input_names) == 2:
```

Comment:
We don't support embedding that has >2 inputs. Instead of checking length, it'll be better to just assert with a proper message when input is different than 2.

Reply:
Synced offline